components/common/Grid.vue

```diff
@@ -0,0 +1,265 @@
+<template>
+  <div class="container">
+    <ul class="grid" :class="{ 'grid--three-cols': threeCols }">
+      <li v-for="item in items" :key="item.name" class="grid_item item">
+        <a href="/grid" class="item">
+          <div class="item__image">
+            <img
+              :src="
+                require(`~/assets/img/maintainers/${item.src
```

Comment:
💬 This is a bit fragile, I think. Can we use the same approach that we did with the project and project images? // cc @glebapps

Reply:
Can we open a new task/issue to talk about that? So you can specify the solution there, thanks!

Reply:
It was just using the name given in the json directly. You can take a look in [this PR](https://github.com/github/globalmaintainersummit.github.com/pull/36/files) (The `FloatingCard.vue` component). ☺️